setup.py

```diff
@@ -226,6 +226,8 @@ def get_torch_arch_list() -> Set[str]:
 
 if _is_cuda():
     vllm_extension_sources.append("csrc/quantization/awq/gemm_kernels.cu")
+    print("\n\n HERE \n\n")
```

Comment:
Please remember to clean up these temporary debug logs.

Reply:
Removed